x-pack/plugins/apm/common/rules/schema.ts

```diff
@@ -17,6 +17,8 @@ export const errorCountParamsSchema = schema.object({
   environment: schema.string(),
   groupBy: schema.maybe(schema.arrayOf(schema.string())),
   errorGroupingKey: schema.maybe(schema.string()),
+  useKqlFilter: schema.maybe(schema.boolean()),
+  kqlFilter: schema.maybe(schema.string()),
```

Comment:
@benakansara For the threshold rule, we use `filterQuery` as the field name, and here we have it as `kqlFilter`. Shall we use the same name for both rules?